solidity/test/DepositRedemptionTest.js

```diff
@@ -317,6 +317,31 @@ describe("DepositRedemption", async function() {
       expect(events[0].returnValues.value).to.eq.BN(signerFee)
     })
 
+    it("escrows correct amount if deposit is pre-term, redeemer is FRT holder and signerFee is partially escrowed", async () => {
+      const partialFee = new BN(123)
```

Comment:
I wouldn’t hate if this was a different random number within the relevant range every time, to be honest. >0 and <signer fee. Not a blocker, though.